.github/workflows/tests.yml

```diff
@@ -17,9 +17,8 @@ jobs:
       fail-fast: true
       matrix:
         php: [8.1, 8.2]
-        stability: [prefer-lowest, prefer-stable]
```

Comment:
I removed `prefer-lowest` from here to not have to bother about phpunit v9. I don't think it's necessary for Lumen anymore to check all of those lowest versions.

Reply:
If you don't want to bother with PHPUnit 9 just remove it from the `composer.json` file.